src/ui/MainRootWindow.qml

```diff
@@ -521,100 +521,85 @@ ApplicationWindow {
+                height:                     additionalErrorsLabel.contentHeight + _margins
+                visible:                    criticalVehicleMessagePopup.dropMessageIndicatorOnClose
+
+                property real _margins: ScreenTools.defaultFontPixelHeight * 0.25
 
-        //-- Dismiss Vehicle Message
-        QGCColoredImage {
-            id:                 criticalVehicleMessageClose
-            anchors.margins:    ScreenTools.defaultFontPixelHeight * 0.5
-            anchors.top:        parent.top
-            anchors.right:      parent.right
-            width:              ScreenTools.isMobile ? ScreenTools.defaultFontPixelHeight * 1.5 : ScreenTools.defaultFontPixelHeight
-            height:             width
-            sourceSize.height:  width
-            source:             "/res/XDelete.svg"
-            fillMode:           Image.PreserveAspectFit
-            color:              qgcPal.alertText
-            MouseArea {
-                anchors.fill:       parent
-                anchors.margins:    -ScreenTools.defaultFontPixelHeight
-                onClicked: {
-                    criticalVehicleMessagePopup.close()
+                QGCLabel {
+                    id:                 additionalErrorsLabel
+                    anchors.centerIn:   parent
+                    text:               qsTr("Additional errors received")
+                    font.pointSize:     ScreenTools.smallFontPointSize
+                    color:              qgcPal.alertText
                 }
             }
         }
 
-        //-- More text below indicator
-        QGCColoredImage {
-            anchors.margins:    ScreenTools.defaultFontPixelHeight * 0.5
-            anchors.bottom:     parent.bottom
-            anchors.right:      parent.right
-            width:              ScreenTools.isMobile ? ScreenTools.defaultFontPixelHeight * 1.5 : ScreenTools.defaultFontPixelHeight
-            height:             width
-            sourceSize.height:  width
-            source:             "/res/ArrowDown.svg"
-            fillMode:           Image.PreserveAspectFit
-            visible:            criticalVehicleMessageText.lineCount > 5
+        QGCLabel {
+            id:                 criticalVehicleMessageText
+            width:              criticalVehicleMessagePopup.width - ScreenTools.defaultFontPixelHeight
+            anchors.centerIn:   parent
+            wrapMode:           Text.WordWrap
             color:              qgcPal.alertText
-            MouseArea {
-                anchors.fill:   parent
-                onClicked: {
-                    criticalVehicleMessageFlick.flick(0,-500)
+        }
+
+        MouseArea {
+            anchors.fill: parent
+            onClicked: {
+                criticalVehicleMessagePopup.close()
+                console.log("Clicked", criticalVehicleMessagePopup.dropMessageIndicatorOnClose);
```

Comment:
Do we need the `console.log`? Where does the message pop up?

Reply:
Nope. Left over debugging. I'll fix it.